addon/-private/execution_context/helpers.js

```diff
@@ -36,3 +36,37 @@ export function fillElement(selection, content, { selector, pageObjectNode, page
     $selection.val(content);
   }
 }
+
+/**
+ * @private
+ *
+ * Given an element, asserts that element is focusable/blurable
+ *
+ * @param {jQuery} element - jQuery object containing the element(s) to check
+ */
+export function assertFocusable(element, { selector, pageObjectNode, pageObjectKey }) {
+  let $element = $(element);
```

Comment:
I was mainly following the convention in `fillElement` that wraps the selection for safety (I assume)  but I'm ok with dropping the extra wrapping, we are currently always passing in a jquery object

Reply:
Ah, actually in `native-events-context` we do just pass the actual element forward. It would be possible to leave it unwrapped I think, but that's why `fillElement` also wraps the element in jquery. I think it'd be best to be consistent here, but I'm open. Thoughts?

Reply:
`fillElement` has some technical debt: we updated it because it needed to be able to take a native DOM element instead of a jQuery selection, and we intended to go through and clean up the remaining uses for consistency/simplicity so that we only ever pass an `element` and never pass a `$selection`. cc: @ro0gr